helper/Datatypes.php

```diff
@@ -53,21 +56,41 @@ class Datatypes
      */
     public static function decodeFile($binary)
     {
-        
         $returnValue = ['name' => '', 'mime' => '', 'data' => ''];
-        
+
         if (empty($binary) === false) {
```

Comment:
Can you do the opposite in this condition, please?

For example:

```
if (empty($binary)) {
    return $returnValue;
}

// regular implementation... 
```

Trying to keep 1 single return in the method only generates a harder code to read and maintain. It's also not recommended according to [Object Calisthenics rules](https://williamdurand.fr/2013/06/03/object-calisthenics/#1-only-one-level-of-indentation-per-method) as most likely you'll have a code with many levels of indentation.